storage/aptosdb/src/lib.rs

```diff
@@ -1201,7 +1201,7 @@ impl DbReader for AptosDB {
                                 None
                             )
                         } else {
-                            let num_txns = committed_version + 1;
+                            let num_txns = executed_trees.state().base_version.map_or(0, |v| v + 1);
```

Comment:
isn't get_frozen_subtree_hashes going to fail if this exceeds the committed version?

Reply:
This is the real committed_version, the old `committed_version` should be LI version.

Reply:
LI version is the committed version, i guess you mean this is the synced version

Reply:
By LI version I mean the version of the latest persisted LI. This line's version is the latest snapshot version, and now we have the latest txn_info version (synced_version). 
get_frozen_subtree_hashes has 0..synced_version >= latest_snapshot_version.